app/controllers/api/projects.php

```diff
@@ -90,7 +89,27 @@
         }
 
         $projectId = ($projectId == 'unique()') ? ID::unique() : $projectId;
+
+        $backups[0] = ['from' => '4:30', 'to' => '5:30'];
+        $backups[1] = ['from' => '11:30', 'to' => '12:30'];
+        $backups[2] = ['from' => '21:30', 'to' => '22:30'];
```

Comment:
let's not index it by the 0,1,2. 

If we change the order of the env variables, it will get messed up. 

use the database names as keys like so `db_fra1_01` and so on. Please verify the db names